com.unity.render-pipelines.universal/Editor/Converter/RenderPipelineConvertersEditor.cs

```diff
@@ -156,19 +178,45 @@ void OnEnable()
 
         public void CreateGUI()
         {
-            m_SerializedObject = new SerializedObject(this);
-            converterEditorAsset.CloneTree(rootVisualElement);
+            if (m_ConverterContainers.Any())
+            {
+                m_SerializedObject = new SerializedObject(this);
+                converterEditorAsset.CloneTree(rootVisualElement);
 
+                rootVisualElement.Q<DropdownField>("conversionsDropDown").choices = m_ContainerChoices;
+                rootVisualElement.Q<DropdownField>("conversionsDropDown").index = m_ContainerChoiceIndex;
+                m_CurrentContainer = m_Containers[m_ContainerChoiceIndex];
+                ReCreateUI();
+
+                var button = rootVisualElement.Q<Button>("convertButton");
+                button.RegisterCallback<ClickEvent>(Convert);
+                button.SetEnabled(false);
+
+                var initButton = rootVisualElement.Q<Button>("initializeButton");
+                initButton.RegisterCallback<ClickEvent>(InitializeAllActiveConverters);
+            }
+        }
+
+        void ReCreateUI()
```

Comment:
```suggestion
        void RecreateUI()
```
Recreate is one word

Reply:
Done